helix-core/src/main/java/org/apache/helix/cloud/azure/AzureCloudInstanceInformationProcessor.java

```diff
@@ -142,8 +142,16 @@ public AzureCloudInstanceInformation parseCloudInstanceInformation(List<String>
         String vmName = computeNode.path(INSTANCE_NAME).getTextValue();
         String platformFaultDomain = computeNode.path(DOMAIN).getTextValue();
         String vmssName = computeNode.path(INSTANCE_SET_NAME).getValueAsText();
+        String azureTopology = AzureConstants.AZURE_TOPOLOGY;
+        String[] parts = azureTopology.trim().split("/");
+        if (parts.length != 2) {
+          throw new HelixException("Invalid Azure topology definition: " + azureTopology);
+        }
```

Comment:
This is not necessary, right? Unless we offer flexibility for user to customize it.

Reply:
I was thinking whether it's possible the file was accidentally changed.

Reply:
But this is not from the file. This is from Java class AzureConstants.

Reply:
oh yes, I was thinking about my property file. Removed.